integration/oidc/authorization-code/README.md

```diff
@@ -0,0 +1,197 @@
+& az ad app permission grant `
+    --api $webApiApp.appId `
+    --id $webAppClient.appId
+```
+
+You will need the following information for the example configuration:
+
+* The client id. This is the value of `$webAppclient.appId`.
+* The client secret. This is the value of `$password`.
+* The GraphQL API App Id URI. This is the value of `$webApiApp.identifierUris[0]`.
+* The Azure AD Directory ID. Run the following command to get it:
+
+    ```powershell
+    az account show | ConvertFrom-Json | ForEach-Object tenantId
+    ```
+
+## Create a New App Registration in Azure AD (Azure Web Portal)
+To represent this example application that wants to access WorkflowGen's
+GraphQL API, we need to register it to Azure AD and give it access to the API.
+This way, when requesting the GraphQL API resource in the OIDC protocol, you
+will get an access token that is a valid JWT and that can be transmitted to
+the API for authorization.
+
+Keep in mind that the user that will authenticate with this application must
+exists in WorkflowGen.
+
+1. In Azure AD, go to the `App Registrations` section.
+1. Click on the `New application registration` button.
+1. Enter the following information:
+
+    Name: `My aspnet core Web App`
+    Application Type: `Web app / API`
+    Sign-on URL: `http://localhost:8080`
+
+1. When you are ready, click on the `Create` button.
+
+You now have registered this example application. You now need to finish
+configuring it in order to get the auhtentication working properly.
+
+1. Go to the `My aspnet core Web App` application registration page in Azure AD.
+1. Copy the Application ID and save it for later use. This is the `Client Id`
+of the application.
+1. Click on the `Settings` button.
+1. Go to the `Reply URLs` section.
+1. Add `http://localhost:8080/signin` to the list and click on the `Save`
+button.
+1. Return to the sections list and go to the `Keys` section.
+1. In the `Passwords` sub-section, add a line with the following information:
+
+    Description: `secret`
+    Expires: `Never expires`
+    Value: A random Guid (see the following note)
+
+    **About Generating a Guid**
+
+    Open a PowerShell terminal and enter the following command:
+    ```powershell
+    [guid]::NewGuid().ToString()
+    ```
+    Copy the result and enter in the value section of the form.
+
+1. Click on the `Save` button.
+1. Copy the value that has been generated after the save. It is the `Client Secret`
+of this application.
+1. Return to the sections list and go to the `Required permissions` section.
+1. Click on the `Add` button.
+1. Click on the `Select an API` button.
+1. Search for the WorkflowGen's GraphQL API application that you've configured
+prior to this guide, click on it and click on the `Select` button.
+1. In the `Select permissions` section, check all checkboxes and click on the
+`Select` button.
+1. Click on the `Done` button.
+
+You should now see WorkflowGen's GraphQL API in the list of required permissions
```

Comment:
You should now see the WorkflowGen GraphQL API in the list of required permissions